src/map.ts

```diff
@@ -127,7 +127,7 @@ export async function mapLimitWithArgs<T, V>(
   limit: number,
   // tslint:disable-next-line: no-any
   iteratee: (value: T, ...args: any) => Promise<V>,
-  // tslint:disable-next-line: no-any
+  // tslint:disable-next-line: no-any trailing-comma
```

Comment:
i think what you did for the rest is correct - should add the trailing comma. This shouldn't have been mergeable before but i think we have not been great about keeping the CI working for this repo since it runs on totally different infra because of public github

Reply:
I was reading the commit where you altered the tslint rules accidentally, but the point about dropping the `trailing-comma` still stands

Reply:
Im sorry if I didn't understood correctly but, what you are saying is that the `trailing-comma` comment should not be here and that the modified rule in the `tslint` as it was in that commit was good?